packages/wordcount/src/stripSpaces.js

```diff
@@ -10,4 +10,5 @@ export default function( settings, text ) {
 	if ( settings.spaceRegExp ) {
 		return text.replace( settings.spaceRegExp, ' ' );
 	}
+	return text;
```

Comment:
It seems sensible to me. It has me somewhat concerned if there were no unit tests to verify this fallback!

Would be a nice improvement, though fine to consider out of scope.

Reply:
Thanks, @aduth. My first thought when trying to cover this code with tests would be to actually redesign it, which I really wanted to avoid with this PR :)

For example, the possibility for `spaceRegExp` to be undefined or the option to be changed from the outside seem very much in [YAGNI land](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) for me and would consider eliminating it. But it works quite well right now, the public interface is reasonably tested, refactoring/rewriting is on the other side of the cost-benefit line for me.